Makefile

```diff
@@ -169,10 +169,10 @@ dsm-%: local.mk
 	@grep -q "^DEFAULT_TC.*=.*$*.*" local.mk || sed -i "/^DEFAULT_TC =/s/$$/ $*/" local.mk
 
 setup-synocommunity: setup
-	@sed -i -e "s|PUBLISH_URL=.*|PUBLISH_URL=https://api.synocommunity.com|" \
```

Comment:
Why not using a regular expression matching any whitespace characters ?
Something such as `\s*` ?

Reply:
(Or `\\s*` depending on the escaping rules obviously...)

Reply:
Mostly because I don't care enough about this. I want to deploy packages not worry about spaces. You are right that is the correct way to do it of course. I just needed a quick fix at the time.